rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, person, book)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] The constructor of Rental should be modified so Book and Person are set in it to break the many-to-many relationship. See suggestion below;

```suggestion
  def initialize(date, person, book)
    @date = date
    @book = book
    book.rentals << self
    @person = person
    person.rentals << self
  end
```